playbooks/cert_renewal.yml

```diff
@@ -1,53 +1,61 @@
 ---
 # automates step 1 of https://github.com/pulibrary/pul-the-hard-way/blob/main/services/create_ssl_certs.md
-# You must pass two variables and you can pass a third if you need it:
-# 1. Pass the first element of the FQDN as '-e host_name=value'
-# for example: '-e host_name=fun' would generate a cert for fun.princeton.edu
-# 2. Pass the path of the directory on your local machine the key and cert should end up in
-# with no trailing slash
-# for example: '-e cert_path=/home/foo/ssl_certs'
-# 3. If your cert needs one or more Subject Alt Names, pass a third variable for those
-# for example: "-e SAN_name='altname'"
+# in the 'vars:' list:
+# 1. Define your host_name values
+#  The first element of the FQDN is the host_names 'name'. For example:
+#  host_names:
+#    - name: fun
+#  would generate a cert for fun.princeton.edu
+# 2. If your cert needs one or more Subject Alt Names, create a list for those
+# 3. Define 'cert_path' as the directory on your local machine the key and cert should end up in
+#  'cert_path` should not have a trailing slash
+#  for example: cert_path: /home/foo/ssl_certs
 #
-# Full example command:
-# ansible-playbook playbooks/cert_renewal.yml -e host_name=new_site -e cert_path=/home/myuser/ssl_certs -e SAN_name='new_site1'
+# Once your vars list is complete, run the playbook:
+# ansible-playbook playbooks/cert_renewal.yml
 #
 # once you have the private key and CSR, follow the rest of the procedure documented on
 # https://github.com/pulibrary/pul-the-hard-way/blob/main/services/create_ssl_certs.md
 #
 - name: Generate a new cert and key
   hosts: localhost
+  # sample vars
+  # vars:
+  #   host_names:
+  #     - name: fubar
+  #       SAN_names:
+  #         - DNS:foo.princeton.edu
+  #     - name: nuts
+  #     - name: plain
+  #       SAN_names:
+  #         - DNS:nut-free.princeton.edu
+  #         - DNS:allergy-friendly.princeton.edu
+  #   cert_path: /Users/foo/ssl_certs
   tasks:
+  - name: Make a subdirectory
+    file:
```

Comment:
FQRN please

Reply:
Thanks for catching that. Addressed by b2c2f94fdabd468d66426badc6a4d3a33ec6588b.